### ISSUE TITLE
Alt Text Issue on Images selected in the 'Text and Image's block widget

### ISSUE THREAD

**Issue by marcemarc** [umbraco/Umbraco.Uno.Issues#37]
The AltText is set to Umbraco.Web.PublishedCache.NuCache.Property?.Value() on images picked for the 'Text and Image' block widget.

![image](https://user-images.githubusercontent.com/260802/94900818-944e6e80-048d-11eb-8a90-db74823880d9.png)

Reproduction
------------

Create a new 'Text and Images' block widget - pick some images, publish, and look at the alttext is written out for the published image... it appears not to be what it should be!

### Specifics

Uno, Umbraco Version 8.7.0...

### Expected result

I'd expect alternative text for the image to be displayed, or if no alt text is available an empty alt text tag.

### Actual result

The Alt Text is showing the code instead.... which is not helpful for people relying on the alt text to convey meaning!

**Comment by ksuvec** [comment]
Thank you for reporting this. It has been fixed and released today 26-10-2020.
